core/dungeon-masters-guide/items/items-wondrous.xml

```diff
@@ -4085,4 +4085,482 @@
+			<p class="indent">The apparatus floats on water. It can also go underwater to a depth of 900 feet. Below that, the vehicle takes 2d6 bludgeoning damage per minute from pressure.</p>
+			<p class="indent">A creature in the compartment can use an action to move as many as two of the apparatus's levers up or down. After each use, a lever goes back to its neutral position. Each lever, from left to right, functions as shown in the Apparatus of Kwalish Levers table.</p>
+			<h5>Apparatus of Kwalish Levers</h5>
+			<table>
+				<thead>
+					<tr><td>Lever</td><td class="col-40 left">Up</td><td class="col-40 left">Down</td></tr>
+				</thead>
+				<tbody>
+					<tr><td>1</td><td class="left">Legs and tail extend, allowing the apparatus to walk and swim.</td><td class="left">Legs and tail retract, reducing the apparatus's speed to 0 and making it unable to benefit from bonuses to speed.</td></tr>
+					<tr><td>2</td><td class="left">Forward window shutter opens.</td><td class="left">Forward window shutter closes.</td></tr>
+					<tr><td>3</td><td class="left">Side window shutters open (two per side).</td><td class="left">Side window shutters close (two per side).</td></tr>
+					<tr><td>4</td><td class="left">Two claws extend from the front sides of the apparatus.</td><td class="left">The claws retract.</td></tr>
+					<tr><td>5</td><td class="left">Each extended claw makes the following melee weapon attack: +8 to hit, reach 5 ft., one target. Hit: 7 (2d6) bludgeoning damage.</td><td class="left">Each extended claw makes the following melee weapon attack: +8 to hit, reach 5 ft., one target. Hit: The target is grappled (escape DC 15).</td></tr>
+					<tr><td>6</td><td class="left">The apparatus walks or swims forward.</td><td class="left">The apparatus walks or swims backward.</td></tr>
+					<tr><td>7</td><td class="left">The apparatus turns 90 degrees left.</td><td class="left">The apparatus turns 90 degrees right.</td></tr>
+					<tr><td>8</td><td class="left">Eyelike fixtures emit bright light in a 30-foot radius and dim light for an additional 30 feet.</td><td class="left">The light turns off.</td></tr>
+					<tr><td>9</td><td class="left">The apparatus sinks as much as 20 feet in liquid.</td><td class="left">The apparatus rises up to 20 feet in liquid.</td></tr>
+					<tr><td>10</td><td class="left">The rear hatch unseals and opens.</td><td class="left">The rear hatch closes and seals.</td></tr>
+				</tbody>
+			</table>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="500">500 lb.</set>
+			<set name="type">Wondrous Items</set>
+			<set name="rarity">Legendary</set>
+		</setters>
+	</element>
```

Comment:
is a duplicate of an element. Apparatus of Kwalish is already in the file at lines 2665-2701. Though weirdly in the description it refers to the item as "Apparatus of the Crab"